libmariadb/ma_charset.c

```diff
@@ -1,27 +1,27 @@
 /****************************************************************************
    Copyright (C) 2012 Monty Program AB
-   
+
```

Comment:
It is undesirable to add insignificant changes to the downstream project. It is more appropriate to include them into the upstream and pull down here with updating a base upstream version.

The reason is that changes of this kind (if we'll have considerable amount of them) may complicate pulldowns from the upstream project.